src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -33,14 +33,12 @@
 #include <mlpack/methods/ann/activation_functions/poisson1_function.hpp>
 #include <mlpack/methods/ann/activation_functions/gaussian_function.hpp>
 
-#include <boost/test/unit_test.hpp>
-#include "test_tools.hpp"
```

Comment:
shouldn't we add `#include "test_catch_tools.hpp"` here ?

Reply:
You're right, thanks!  0d651e29